mitosheet/mitosheet/step_performers/step_performer.py

```diff
@@ -102,6 +105,7 @@ def execute_through_transpile(
         execution_data: Optional[Dict[str, Any]]=None,
         new_dataframe_params: Optional[ExecuteThroughTranspileNewDataframeParams]=None,
         column_headers_to_column_ids: Optional[Dict[ColumnHeader, ColumnID]]=None,
+        optional_code: Optional[Tuple[List[str], List[str]]]=None,
```

Comment:
See https://github.com/mito-ds/mito/issues/1110

Reply:
This is fixed in #1115